test/gpu/jit.cpp

```diff
@@ -35,11 +35,14 @@
 #include <migraphx/gpu/compile_hip.hpp>
 #include <migraphx/gpu/compile_hip_code_object.hpp>
 #include <migraphx/gpu/compiler.hpp>
+#include <migraphx_kernels.hpp>
 
 // NOLINTNEXTLINE
 const std::string write_2s = R"__migraphx__(
 #include <hip/hip_runtime.h>
+#include <migraphx/kernels/types.hpp>
 
+using namespace migraphx;
```

Comment:
Just used `char` and skip including migraphx header for this test,

Reply:
Done